tools/src/publish/types.ts

```diff
@@ -0,0 +1,13 @@
+export type PublishOptions = {
+  projectName?: string;
+  directory?: string;
+  registry?: string;
+  tag?: string;
+  nextVersion: string;
+  verbose: boolean;
```

Comment:
```suggestion
  verbose?: boolean;
```

Otherwise there's a type error in `global-setup.e2e.ts`:

![image](https://github.com/user-attachments/assets/0cd0b6b4-67b5-473c-a061-88faf7a8de9d)

Reply:
made optional